crates/spacewalk/Cargo.toml

```diff
@@ -60,7 +61,7 @@ std = [
   "sp-std/std",
   'orml-traits/std',
   'base64/std',
-  'substrate-stellar-sdk/std',
+  'stellar-support/std',
```

Comment:
there are 2 `substrate-stellar-sdk/std` defined. The first one is found at like 53.